app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
Using a class attribute `people` to store instances of `Person` can lead to potential issues, especially in a multi-threaded environment or when multiple instances of `Person` are created and modified. Consider using instance attributes or another design pattern to manage relationships between `Person` instances.